components/common/ConnectWallet/ConnectWalletModal.vue

```diff
@@ -18,58 +18,94 @@
           v-for="(wallet, index) in installedWallet"
           :key="index"
           :wallet="wallet"
-          @setAccount="setAccount" />
+          @set-account="setAccount" />
+      </div>
+
+      <div v-if="showUninstalledWallet" class="!px-8 !pt-6">
+        <div class="font-bold capitalize mb-3">
+          {{ $i18n.t('walletConnect.newToKoda') }}
+        </div>
+        <div class="text-xs !mb-6">
+          {{ $i18n.t('walletConnect.startConnect') }}
+        </div>
+        <div class="border py-4 !px-6 !mb-6 bg-k-grey-light border-k-grey">
+          <div class="mb-3">{{ $i18n.t('walletConnect.quickSteps') }}</div>
+          <ol class="text-xs font-bold pl-4">
+            <li class="mb-3">
+              {{ $i18n.t('walletConnect.chooseWallet')
+              }}<span class="font-normal">{{
+                $i18n.t('walletConnect.chooseBelow')
+              }}</span>
+            </li>
+            <li>
+              {{ $i18n.t('walletConnect.downloadAndConnect')
+              }}<span class="font-normal">{{
+                $i18n.t('walletConnect.setUpConnect')
+              }}</span>
+            </li>
+          </ol>
+        </div>
+        <div class="font-bold mb-3 capitalize">
+          {{ $i18n.t('walletConnect.selectWallet') }}
+        </div>
```

Comment:
maybe can use gap
https://tailwindcss.com/docs/gap#setting-the-gap-between-elements

in order to reduce some of the `mb-3` `mb-6`

?

Reply:
I would not use `gap` here, because these items' gaps are not the same according to the design. It would not reduce much.